crates/godwoken-bin/src/main.rs

```diff
@@ -50,32 +53,32 @@ fn generate_example_config<P: AsRef<Path>>(path: P) -> Result<()> {
 // TODO: @zeroqn update clap to v3
 async fn run_cli() -> Result<()> {
     let version = Version::current().to_string();
-    let app = App::new("Godwoken")
+    let app = Command::new("Godwoken")
```

Comment:
`App` and `SubCommand` are deprecated in clap 3 and removed in clap 4. We should gradually remove usage of deprecated clap 3 features so we can eventually upgrade to clap 4.